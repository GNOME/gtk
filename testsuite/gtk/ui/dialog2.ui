<!-- missing attribute in dialog markup -->
<interface>
  <object class="GtkDialog">
    <child type="action">
      <object class="GtkButton" id="button_ok">
      </object>
    </child>
    <action-widgets>
      <action-widget>button_ok</action-widget>
    </action-widgets>
  </object>
</interface>
