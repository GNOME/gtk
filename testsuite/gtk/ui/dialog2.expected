ERROR: g-markup-error-quark 6
.:9:23 element 'action-widget' requires attribute 'response'
